### PR TITLE
Fix CHANGELOG for recent 0.7 release

### PR DESCRIPTION
**Pull request by adamgreig**
:sweat_smile:

### CODE REVIEW

**Review by thejpster — APPROVED** [review]

**Comment by adamgreig** [comment]
bors r=thejpster

(btw, @thejpster, feel free to `bors r+` if you're happy and the PR's not marked as a draft, or otherwise comment if you'd like someone else on the team to check it before merging)

**Comment by thejpster** [comment]
Ok, sure. I forget you have bors here - usually my repos don't.

File: CHANGELOG.md [hunk 1]
```diff
@@ -570,7 +572,8 @@ section                size        addr
 
 Initial release
 
-[Unreleased]: https://github.com/rust-embedded/cortex-m-rt/compare/v0.6.11...HEAD
+[Unreleased]: https://github.com/rust-embedded/cortex-m-rt/compare/v0.7.0...HEAD
+[v0.7.0]: https://github.com/rust-embedded/cortex-m-rt/compare/v0.6.11...v0.7.0
```
Comment (therealprof) [inline CHANGELOG.md:576]: Is the lower version correct?

Comment (adamgreig) [reply CHANGELOG.md:576]: Yep, that's when we branched off to the separate 0.6 branch; 0.6.12 and onwards were all backports from master.

**Pull request merged**